src/re_frame_request/events.cljs

```diff
@@ -106,32 +127,36 @@
     :keys [name
            on-success
            on-failure
+           on-abort
            on-progress]
     :or {on-success [:http-no-on-success]
          on-failure [:http-no-on-failure]}}]
   (try
-    (let [seq-request-maps (if (sequential? request) request [request])
```

Comment:
Not sure how `request` could have been `sequential?` as the destructuring in the function args indicates it's a map, and therefore `name`, `on-success` etc would be missing